lang/java/avro/src/test/java/org/apache/avro/TestSchema.java

```diff
@@ -421,6 +426,53 @@ void qualifiedName() {
     assertEquals("Int", nameInt.getQualified("space"));
   }
 
+  @Test
+  void validValue() {
```

Comment:
Do we want to explicitly mention we're testing default values here?
(a matter of style/opinion only)